pandas/tests/io/test_pickle.py

```diff
@@ -299,6 +300,15 @@ def test_pickle_v0_15_2():
     tm.assert_categorical_equal(cat, pd.read_pickle(pickle_path))
 
 
+def test_pickle_path_pathlib():
+    tm._skip_if_no_pathlib()
```

Comment:
yep exactly like this!

maybe put a little docs in the testing file above these 3 tests routines.